src/libstd/time.rs

```diff
@@ -60,6 +60,22 @@ pub use core::time::Duration;
 /// }
 /// ```
 ///
+/// # OS-specific behaviours
```

Comment:
```suggestion
/// # OS-specific behaviors
```

It looks like we prefer American English over British English:

```
src % rg --type rust 'behaviour' | wc
      51     560    5421
src % rg --type rust 'behavior' | wc
     685    6956   67377
```